spec/plugins/data_table/server_side.js

```diff
@@ -433,4 +434,57 @@ describe("DataTable Plugin", function() {
     });
   });
 
+  describe("filtering", function() {
+    beforeEach(function() {
+      table = new app.Views.T({ collection : collection });
+      table.render();
+      jasmine.Ajax.requests.mostRecent().response(mockResponse.get());
+    });
+
+    it("should have filtering info for filterable column in column manager", function() {
```

Comment:
"should contain an object that describes the type of filter to be applied and possible values if applicable." or something more descriptive.